src/components/pinnable.js

```diff
@@ -51,7 +51,7 @@ AFRAME.registerComponent("pinnable", {
         easing: "easeOutElastic"
       });
 
-      if (this.el.components["body-helper"]) {
+      if (this.el.components["body-helper"] && !this.el.sceneEl.systems.interaction.isHeld(this.el)) {
```

Comment:
While holding an object, I expect to be able to move it until I drop it. This makes it so that pinning an object while holding it doesn't cause it to freeze in place.